docs/undergrad.yaml

```diff
@@ -508,8 +506,10 @@ Measures and integral Calculus:
     integrable functions: 'measure_theory.integrable'
     dominated convergence theorem: 'measure_theory.tendsto_integral_of_dominated_convergence'
     finite dimensional vector-valued integrable functions: 'measure_theory.integrable'
-    continuity of integrals with respect to parameters:
-    differentiability of integrals with respect to parameters:
+    continuity of integrals with respect to parameters: 'interval_integral.continuous_on_integral_of_continuous'
+    differentiability of integrals with respect to parameters: 'interval_integral.differentiable_on_integral_of_continuous'
+    fundamental theorem of calculus, part 1: 'interval_integral.integral_has_fderiv_at_of_tendsto_ae'
+    fundamental theorem of calculus, part 2: 'interval_integral.integral_eq_sub_of_has_deriv_right_of_le'
```

Comment:
Is this numbering really standard in English? I never encountered this before Yury started to use it on Zulip. I would call the first one "existence of antiderivative" or "existence of primitive" and the second one "integral of a derivative function". By the way, there is a line "antiderivative" somewhere in this file, so we have a conflict here. Strangely enough I wasn't able to find a line which was waiting for the second one.

Reply:
Yes, referring to them by number is standard (although I always have to look up which is which!)

Reply:
But on the first pass I missed that this was a new line.  I don't think we should add lines to `undergrad.yaml`, I think it should be kept in exact match with the French curriculum source.  And indeed, there seems to be no line for FTC-2 in the original, only FTC-1:

> **Intégrale sur un segment des fonctions continues par morceaux**
Calcul de primitives. Sommes de Riemann. Primitives d’une fonction continue. Méthodes usuelles de calcul d’intégrales. Changement de variable. Intégration par parties.

so I guess this list should also omit FTC-2.

To reduce friction I'll integrate the fixes at #5675, let's continue the discussion there if needed.